packages/manager/apps/container/src/container/nav-reshuffle/sidebar/navigation-tree/services/securityIdentityOperation.ts

```diff
@@ -6,24 +8,45 @@ export default {
     application: 'iam',
   },
   count: false,
-  features: ['iam'],
+  features: ['iam', 'key-management-service'],
   children: [
     {
-      id: 'security-identity-operation-iam',
-      translation: 'sidebar_security_identity_operations_iam',
-      routing: {
-        application: 'iam',
-        hash: '#/',
-      },
+      id: 'security-identity',
+      translation: 'sidebar_security_identity',
+      children: [
+        {
+          id: 'security-identity-operation-iam',
+          translation: 'sidebar_security_identity_operations_iam',
+          routing: {
+            application: 'iam',
+            hash: '#/',
+          },
+        },
+        {
+          id: 'security_identity_operations_kms',
+          translation: 'sidebar_security_identity_operations_kms',
+          features: ['key-management-service'],
+          routing: {
+            application: 'key-management-service',
+            hash: '#/',
+          },
+        },
+      ]
     },
     {
-      id: 'security-identity-operation-logs',
-      translation: 'sidebar_security_identity_operations_logs',
-      serviceType: 'DBAAS_LOGS',
-      routing: {
-        application: 'dedicated',
-        hash: '#/dbaas/logs',
-      },
-    },
+      id: 'security-operations',
+      translation: 'sidebar_security_operations',
+      children: [
+        {
+          id: 'security-identity-operation-logs',
+          translation: 'sidebar_security_identity_operations_logs',
+          serviceType: 'DBAAS_LOGS',
+          routing: {
+            application: 'dedicated',
+            hash: '#/dbaas/logs',
+          },
+        }
```

Comment:
Doesn't it need a features property ?
On the classical nav it has
```
['logs-data-platform']
```
(cf: https://github.com/ovh/manager/blob/master/packages/manager/apps/container/src/container/legacy/server-sidebar/universe/DedicatedSidebar.tsx#L238C16-L238C38)